### DISCUSSION TITLE
Multiple area selection feature to crop or get left/right/top/bottom coordinates

### DISCUSSION THREAD

**Discussion by giri-ggr** [advanced-cropper/vue-advanced-cropper#189] [Q&A]
I am looking at multiple area selections in the "vue-advanced-cropper" but I did not find it, is it possible?

I have attached a screenshot have a look once.

![WhatsApp Image 2022-05-03 at 10 07 32 PM](https://user-images.githubusercontent.com/65165187/166499748-c63c4de6-7413-488e-80cf-817c86dea0c4.jpeg)

**Comment by Norserium** [comment]
@giri-ggr , it's impossible. 

The previous discussions on this theme: [[1]](https://github.com/Norserium/vue-advanced-cropper/issues/50#issuecomment-675413923), [[2]](https://github.com/Norserium/vue-advanced-cropper/discussions/162). The main point is:
> If it were possible it would be the completely different cropper. [...] I don't see that it will be possible in this library ever.

It's pretty rare use-case to make this library more complex than it is now (and it's too much complex now already). I can advise you try to use the alternative cropper. 

For example, [`JCrop`](https://jcrop.com/examples/custom-widget.html).

**Reply by giri-ggr** [reply]
Thanks for the response. Actually, I previously had looked into a package(https://jcrop.com/examples/custom-widget.html) that used to allow this, but I also did require features from this package, so continued with this one.

**Reply by Norserium** [reply]
@giri-ggr, I'm sorry that this library is not helpful you in your situation. But I don't see any acceptable ways to extend it to implement this feature. It's better to create the independent library, probable even on this library foundation, but, alas, I don't have enough time for it, so the only way now is looking for alternatives.
